src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1009 @@
+			if (tree.SelectionMode == TreeViewSelectionMode.Single && result.Count > 0)
+			{
+				if (tree.SelectedItem != tree.SelectedItems[0] || tree.SelectedNode != tree.SelectedNodes[0])
+				{
+					return "SelectedItem!=SelectedItems[0] || SelectedNode!=SelectedNodes[0]";
+				}
+			}
+
+			return result.Count > 0 ? "Selected: " + string.Join(", ", result) : "Nothing selected";
+		}
+
+		private void GetSelected_Click(object sender, RoutedEventArgs e)
+		{
+			if (IsInContentMode())
+			{
+				Results.Text = GetSelection(ContentModeTestTreeView);
+			}
+			else
+			{
+				Results.Text = GetSelection(TestTreeView);
+			}
+		}
+
+		private void TestTreeView_ItemInvoked(TreeView sender, TreeViewItemInvokedEventArgs args)
+		{
+			var data = ((TreeViewNode)args.InvokedItem).Content.ToString();
+			Results.Text = "ItemClicked:" + data;
+
+			if (!_disableClickToExpand)
+			{
+				var node = args.InvokedItem as TreeViewNode;
+				node.IsExpanded = !node.IsExpanded;
+			}
+		}
+
+		private void TestTreeViewContentMode_ItemInvoked(TreeView sender, TreeViewItemInvokedEventArgs args)
+		{
+			var item = args.InvokedItem as TreeViewItemSource;
+			Results.Text = "ItemClicked:" + item.Content;
+
+			if (!_disableClickToExpand)
+			{
+				item.IsExpanded = !item.IsExpanded;
+			}
+		}
+
+		private void MoveNodesToNewTreeView_Click(object sender, RoutedEventArgs e)
+		{
+			if (IsInContentMode())
+			{
+				ContentModeTestTreeView.ItemsSource = null;
+				TestTreeView2ItemsSource.Add(TestTreeViewItemsSource[0]);
+				ContentModeTestStackPanel.Children.Remove(ContentModeTestTreeView);
+			}
+			else if (_visualRoot != null)
+			{
+				TestTreeView.RootNodes.Remove(_visualRoot);
+				TestTreeView2.RootNodes.Add(_visualRoot);
+				TestStackPanel.Children.Remove(TestTreeView);
+			}
+		}
+
+		private void DisableClickToExpand_Click(object sender, RoutedEventArgs e)
+		{
+			_disableClickToExpand = true;
+		}
+
+		private void SetupExpandingNodeEvent_Click(object sender, RoutedEventArgs e)
+		{
+			if (IsInContentMode())
+			{
+				TreeViewItemSource item = new TreeViewItemSource() { Content = "Virtualized", HasUnrealizedChildren = true };
+				TestTreeViewItemsSource[0].Children.Add(item);
+				ContentModeTestTreeView.Expanding += ContentModeTestTreeView_Expanding;
+			}
+			else if (_visualRoot != null)
+			{
+				_virtualizedNode = new TreeViewNode() { Content = "Virtualized" };
+				_virtualizedNode.HasUnrealizedChildren = true;
+				_visualRoot.Children.Add(_virtualizedNode);
+
+				TestTreeView.Expanding += TestTreeView_Expanding;
+			}
+		}
+
+		private void LabelItems_Click(object sender, RoutedEventArgs e)
+		{
+			if (IsInContentMode())
+			{
+				SetAutomationIdForNodes(ContentModeTestTreeView);
+			}
+			else
+			{
+				SetAutomationIdForNodes(TestTreeView);
+			}
+		}
+
+		private void SetAutomationIdForNodes(TreeView tree)
+		{
+			var listControl = VisualTreeUtils.FindVisualChildByName(this.TestTreeView, "ListControl") as TreeViewList;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'listControl'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)